hack/jenkins/e2e.sh

```diff
@@ -74,6 +74,11 @@ if [[ ${JOB_NAME} =~ ^kubernetes-.*-gce ]]; then
 elif [[ ${JOB_NAME} =~ ^kubernetes-.*-gke ]]; then
   KUBERNETES_PROVIDER="gke"
   : ${E2E_ZONE:="us-central1-f"}
+elif [[ ${JOB_NAME} =~ ^kubernetes-.*-aws ]]; then
+  KUBERNETES_PROVIDER="aws"
+  : ${E2E_MIN_STARTUP_PODS:="1"}
+#  : ${E2E_ZONE:="us-central1-f"}
```

Comment:
I think this line slipped in. If the user will need to set up the ```E2E_ZONE``` but uncommenting and filling in the field I recommend either setting a sane default which can be overridden or erroring out noting it's required.